src/main/java/edu/iis/mto/similarity/Search.java

```diff
@@ -0,0 +1,27 @@
+package edu.iis.mto.similarity;
+
+import edu.iis.mto.searcher.SearchResult;
+import edu.iis.mto.searcher.SequenceSearcher;
+
+public class Search implements SequenceSearcher {
+
+    SearchResult sr = SearchResult.builder().withFound(false).withPosition(-1).build();
+
+    private int calls=0;
+
+    @Override
+    public SearchResult search(int elem, int[] sequence) {
+        calls++;
+        for (int i : sequence){
```

Comment:
nie chodziło o to, żeby samodzielnie implementować algorytm - to miał być "mock"